### PR TITLE
[WIP] Package dephell

### PR DESCRIPTION
**Pull request by chutz**
This is a work in progress (still some dependencies missing).
Packaging https://github.com/dephell/dephell to convert pyproject.toml to setup.py

### CODE REVIEW

**Comment by mgorny** [comment]
As I said on IRC, I'm not sure if it wouldn't be both less work right now and less maintenance effort in the future if we made a trivial in-house pyproject2setup.py converted suitable for our needs. Some people may also be thankful for not having all those deps installed.

File: dev-python/ruamel-yaml/ruamel-yaml-0.16.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+SRC_URI="https://bitbucket.org/${MY_PN/.//}/get/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="test +clib"
+
+BDEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	clib? ( dev-python/ruamel-yaml-clib[${PYTHON_USEDEP}] )
```
Comment (andrey-utkin) [inline dev-python/ruamel-yaml/ruamel-yaml-0.16.5.ebuild]: Would be great to have the real change separated from ebuild file cloning into standalone commit.
Without that, I recognise that this addition of a USE flag is the most important change.
Could you please elaborate the reasoning behind this change, in commit message please?

I don't see any change to configure and compile stages, so we must be ending up with one of
* USE flag for optional runtime dependency, which is bad;
* package's dependency discovery doing its uncontrolled automagic, which is bad.

Comment (orsinium) [reply dev-python/ruamel-yaml/ruamel-yaml-0.16.5.ebuild]: Dephell supports both ruamel.yaml and PyYAML. So, if installing the first one is tricky, just use the second one.

**Comment by mkszuba** [comment]
For the record, latest versions of dev-python/ruamel-yaml{,-clib} available in Gentoo now are newer than those in this PR.

**Pull request closed without merging**